js/components/file.js

```diff
@@ -57,22 +57,31 @@ Fliplet.FormBuilder.field('file', {
     },
     onReset: function() {
       var $vm = this;
-      
+
       $vm.value = [];
       $vm.selectedFileName = '';
-      
+
       $vm.$emit('_input', $vm.name, $vm.value);
     },
+    ensureValue: function () {
```

Comment:
What does it ensure the value to be?

You could either call it `validateValue()`, which is what it's doing.

Reply:
Agreed and renamed 👍